matplotcheck/raster.py

```diff
@@ -140,27 +184,12 @@ def assert_legend_accuracy_classified_image(
         # Check that each legend entry label is in one of all_label_options
         assert len([val for val in legend_dict.values() if val]) == len(
```

Comment:
Here it looks like you're testing that the labels occur in the legend? it seems like this helper should just return the labels in the legend for the plot.